### PR TITLE
Fixed searchString bug. Removed nested IOC mapping structure.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
### Description
Fixed searchString bug. Removed nested IOC mapping structure.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/main/java/org/opensearch/securityanalytics/transport/TransportListIOCsAction.java [hunk 1]
```diff
@@ -193,21 +191,12 @@ private void listIocs(List<String> iocIndices) {
                 boolQueryBuilder.must(
                         QueryBuilders.queryStringQuery(request.getTable().getSearchString())
                                 .defaultOperator(Operator.OR)
-//                            .field(STIX2_IOC_NESTED_PATH + STIX2IOC.ID_FIELD) // Currently not a column in UX table
```
Comment (eirsep) [inline src/main/java/org/opensearch/securityanalytics/transport/TransportListIOCsAction.java:196]: Is default operator required?

**Comment by eirsep** [comment]
plz ensure you chekc threat intel monitor flow in integ tests where we query ioc indices while scanning user data for malicious IoCs

File: src/main/java/org/opensearch/securityanalytics/transport/TransportListIOCsAction.java (hunk 1 above)
Comment (AWSHurneyt) [reply src/main/java/org/opensearch/securityanalytics/transport/TransportListIOCsAction.java:196]: Good catch. The default operator is already `OR`, so this is redundant. Will revise.

**Comment by AWSHurneyt** [comment]
> plz ensure you chekc threat intel monitor flow in integ tests where we query ioc indices while scanning user data for malicious IoCs

Troubleshooting the integ failures now. I had them running locally, but wanted to get these changes out for early feedback.

**Review by eirsep — APPROVED** [review]

**Review by sbcd90 — APPROVED** [review]

**Review by amsiglan — APPROVED** [review]

**Pull request merged**